libs/designsystem/page/src/page.component.ts

```diff
@@ -689,6 +718,24 @@ export class PageComponent
     this.isObservingTitle = false;
   }
 
+  private setHtmlDocTitle() {
+    if (!this.config?.setHtmlDocTitle) return;
+
+    const toolbar: HTMLElement = this.ionHeaderElement.nativeElement.querySelector('ion-toolbar');
+
+    const titleArea: HTMLElement = toolbar.querySelector('ion-title .toolbar-title');
```

Comment:
Could this be achieved with only one query selector?

Reply:
Very much 🙏 Changed it to one selector.